### ISSUE TITLE
Known alternatives to SE

### ISSUE THREAD

**Issue by SoniEx2** [shinigami-eyes/shinigami-eyes#89]
Since the extension developers have basically gone rogue, we're compiling a list of known alternatives of varying quality to this extension.

Currently known alternatives:

- [SELF-PLUG] https://github.com/gd-irc/Transmisia - Fully decentralized, unlikely to get banned in Norway.

- - -

Feel free to suggest more!

**Comment by NovaMoon** [comment]
> basically gone rogue
i can't really find anything related to this claim. just that the extension is apparently abandoned.

**Comment by SoniEx2** [comment]
yeah except for the silent closed-source updates to the extension in the chrome web store etc?

**Comment by 0xallie** [comment]
https://github.com/bethylamine/soupcan

Twitter only, but incorporates Shinigami Eyes's bloom filters and any account you encounter that's flagged by Shinigami Eyes gets automatically reported for addition to Soupcan's database.
